paderbox/array/interval/core.py

```diff
@@ -358,6 +358,42 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
```

Comment:
Why is this a property? `as_something` is often a method (e.g., `dataclasses.as_dict`) and could be confusing if this one is a property.

Reply:
Or call this `to_serializable` as in `np.ndarray.to_list`?

Reply:
I wanted to keep it as similar as possible to the property `_intervals_to_str`. I renamed it to `to_serializable`, now.

Reply:
Do you also remove the `@property`? The `@property` for `_intervals_to_str` is only ok, because it is private.

Reply:
Done